donkeycar/parts/launch.py

```diff
@@ -46,3 +48,73 @@ def run(self, mode, ai_throttle):
 
         return new_throttle
 
+
+class AiCatapult():
```

Comment:
This class is effectively just a copy of the above class with a very small change to return 0 angle in launch mode. We should probably drive this with a single bool switch in the `AiLaunch` class above. Also the `()` should go. Then we should add a config parameter in `cfg_complete.py` that can trigger the 0 angle behaviour and pass this in `complete.py`